src/Decompiler/Scanning/BlockConflictResolver.cs

```diff
@@ -220,6 +220,49 @@ private void RemoveBlockFromGraph(RtlBlock n)
         {
             //Debug.Print("Removing block: {0}", n.Address);
             blocks.Nodes.Remove(n);
+            foreach (var i in n.Instructions)
+            {
+                RemoveDirectlyCalledAddress(i);
+            }
+        }
+
+        private void RemoveDirectlyCalledAddress(RtlInstructionCluster i)
+        {
+            if ((i.Class & InstrClass.Call) == 0)
+                return;
+            if ((i.Class & InstrClass.Transfer) == 0)
+                return;
```

Comment:
This could be made slightly more clear:
```C#
if ((i.Class & (InstrClass.Call|InstrClass.Transfer)) != (InstrClass.Call|InstrClass.Transfer))
```

Reply:
I have changed it to
```
    var callTransfer = InstrClass.Call | InstrClass.Transfer;
    if ((i.Class & callTransfer) != callTransfer)
```